pkg/daemon/daemon.go

```diff
@@ -195,38 +172,27 @@ func New(
 
 // Run the config daemon
 func (dn *Daemon) Run(stopCh <-chan struct{}, exitCh <-chan error) error {
-	log.Log.V(0).Info("Run()", "node", dn.name)
+	log.Log.V(0).Info("Run()", "node", vars.NodeName)
 
-	if utils.ClusterType == utils.ClusterTypeOpenshift {
-		log.Log.V(0).Info("Run(): start daemon.", "openshiftFlavor", dn.openshiftContext.OpenshiftFlavor)
+	if vars.ClusterType == consts.ClusterTypeOpenshift {
+		log.Log.V(0).Info("Run(): start daemon.", "openshiftFlavor", dn.platformHelpers.GetFlavor())
 	} else {
 		log.Log.V(0).Info("Run(): start daemon.")
 	}
 
-	if dn.useSystemdService {
+	if !vars.UsingSystemdMode {
 		log.Log.V(0).Info("Run(): daemon running in systemd mode")
```

Comment:
log is now wrong. maybe change to : "daemon running in daemon mode" and add an else statement with a log that says "daemon running in systemd mode"

Reply:
right!